packages/frontend/src/features/sqlRunner/store/sqlRunnerSlice.ts

```diff
@@ -56,10 +56,11 @@ export const sqlRunnerSlice = createSlice({
         setProjectUuid: (state, action: PayloadAction<string>) => {
             state.projectUuid = action.payload;
         },
-        setInitialResultsTableConfig: (
+        setInitialResultsAndSeries: (
```

Comment:
I combined the chart and table reducers here to avoid a double-dispatch. There are other ways to do this if you'r prefer.